File: Servicios/Empleado/Empleado.cs
namespace ChatSignalR.Servicios.Empleado
{
    public class Empleado : EntityBase.Base
    {
        public int Legajo { get; set; }
        public string Nombre {get; set; }
        public string Apellido { get; set; }
        public string Dni { get; set; }
        public string ApyNom => $"{Apellido}, {Nombre}";
        
    }
}

File: wwwroot/ScriptsTS/EmpleadoTS/Empleado.ts
export class Empleado
{
    legajo : number;
    nombre : string;
    apellido : string;
    dni : string;
    apyNom : string;

    constructor(legajo : number = null, nombre : string = null, apellido : string = null, dni : string = null)
    {
        this.legajo = legajo;
        this.nombre = nombre;
        this.apellido = apellido;
        this.dni = dni;
    }
}